deployer.rb

```diff
@@ -82,8 +72,21 @@ def repo_name(pull_request)
     pull_request['base']['repo']['full_name']
   end
 
-  def build_status
-    "success"
+  def pull_request_number(pull_request)
+    pull_request["number"]
+  end
+
+  def build_status(pull_request)
+    leankit_ticket_present?(pull_request) ? "success" : "error"
+  end
+
+  def description(pull_request)
+    d = leankit_ticket_present?(pull_request) ?
+      slack_channel_topic :
+      "None of the messages in the PR include LeanKit ID"
+
+    puts d
```

Comment:
Did you mean for this to print to be here?

Reply:
Yeah, i want to keep it there for debugging purproses so i have something to look for in the logs associated with API calls